src/lib/dom.generated.d.ts

```diff
@@ -12633,6 +12644,14 @@ declare var HTMLTemplateElement: {
     new(): HTMLTemplateElement;
 }
 
+interface HTMLPictureElement extends HTMLElement {
+}

```

Comment:
That kind of stinks, but I guess there's nothing we can really add as far as I see.